src/gov-action-contracts/util/OfficeHoursAction.sol

```diff
@@ -0,0 +1,34 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity 0.8.16;
+
+/// @notice Action that requires the current time to be within office hours. Should be included as an L1 action in proposal data.
+/// @dev    Time is kept in UTC
+contract OfficeHoursAction {
+    uint256 public immutable minUtcHour;
+    uint256 public immutable maxUtcHour;
+    uint256 public immutable minimumTimestamp;
+
+    constructor(uint256 _minUtcHour, uint256 _maxUtcHour, uint256 _minimumTimestamp) {
+        require(_minUtcHour < _maxUtcHour, "Invalid office hours");
+        require(_minUtcHour < 24 && _maxUtcHour < 24, "Invalid office hours");
```

Comment:
this limit the action to be used on hours that crossed the 24 hour mark
so this mean it is not possible to have end hour after `EST 7pm` or `JST 9am` (both are UTC 2500)